go.mod

```diff
@@ -3,41 +3,45 @@ module github.com/vmware-tanzu/velero-plugin-for-microsoft-azure
 go 1.18
 
 require (
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.6.0
-	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.0
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.7.1
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute v1.0.0
-	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/storage/armstorage v1.2.0
-	github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v0.6.1
+	github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v1.1.0
 	github.com/gofrs/uuid v4.3.1+incompatible
-	github.com/joho/godotenv v1.4.0
 	github.com/pkg/errors v0.9.1
-	github.com/sirupsen/logrus v1.9.0
+	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.2
-	github.com/vmware-tanzu/velero v0.0.0-20230727074327-a6d79fc272a2
+	github.com/stretchr/testify v1.8.4
+	github.com/vmware-tanzu/velero v1.10.0-rc.1.0.20230920021852-d3e5bb745144
```

Comment:
Is there a reason to reference a Velero RC tag?

Reply:
This is generated by go cmd automatically when running `go get github.com/vmware-tanzu/velero@main`.
The tag isn't important as this is a pseudo version